{{cookiecutter.project_slug}}/{{cookiecutter.package_name}}/jinja2.py

```diff
@@ -1,6 +1,20 @@
 import jinja2
 from django.templatetags.static import static
 from django.urls import reverse
+from markdown_it import MarkdownIt
+
+markdown_converter = MarkdownIt()
+
+
+def url(path, *args, **kwargs):
+    assert not (args and kwargs)
+    return reverse(path, args=args, kwargs=kwargs)
+
+
+def markdown(text, cls=None):
+    html = markdown_converter.render(text).strip()
+    html = html.replace("<p>", f'<p class="{cls or ""}">', 1).replace("</p>", "", 1)
```

Comment:
What is this supposed to be doing?  We probably need a docstring or comment in here to explain (or remove this line)